packages/build/tests/plugins_list/tests.js

```diff
@@ -26,12 +25,21 @@ const runWithApiMock = async function (
         ...flags,
       })
       .runWithBuild()
-    await t.snapshot(normalizeOutput(output))
+    return normalizeOutput(output)
   } finally {
     await stopServer()
   }
 }
 
+const runWithApiMock = async function (
+  t,
+  fixtureName,
+  { testPlugin, response = getPluginsList(testPlugin), ...flags } = {},
+  status = 200,
+) {
+  await t.snapshot(await runWithApiMockAndGetNormalizedOutput(fixtureName, { testPlugin, response, ...flags }, status))
+}
+
```

Comment:
All existing tests in this suite use snapshots which is not something I wanted to continue and wanted something a bit more concrete, so I moved most of this to another helper that just returns normalized output instead of snapshotting it.

Would be nice to convert existing tests to explicit assertions, but I don't have bandwidth for that right now